homeassistant/components/tasmota/discovery.py

```diff
@@ -0,0 +1,170 @@
+"""Support for MQTT discovery."""
+import asyncio
+import json
+import logging
+
+from hatasmota.const import CONF_ID, CONF_RELAY
+from hatasmota.discovery import (
+    TasmotaDiscoveryMsg,
+    get_device_config,
+    get_entities_for_platform,
+    get_entity,
```

Comment:
Please rename these names on import so it's clear it's tasmota entities and not home assistant entities that are meant.